### PR TITLE
chore(agw): activate mypy code scanning in lte except integ_tests

### PR DESCRIPTION
**Pull request by ajahl**
Signed-off-by: Alex Jahl <alexander.jahl@tngtech.com>

## Summary

Active MyPy scanning for all Python code in `lte` except `integ_tests` (is part of a separate PR) and fix all MyPy related errors.

## Test Plan

```
vagrant@magma$ cd $MAGMA_ROOT/lte/gateway
vagrant@magma$ mypy --ignore-missing-imports python/magma/
```

```
vagrant@magma$ bazel test //lte/gateway/python:all
```

- [x] Verified S1AP Integration Tests

## Additional Information

This change is not backwards-breaking

### CODE REVIEW

File: lte/gateway/python/magma/pipelined/encoding.py [hunk 1]
```diff
@@ -27,94 +27,80 @@
...
-        enc = cipher.encrypt(pad(s).encode('utf-8'))
-
-        hmac = HMAC.new(key_mac)
-        hmac.update(iv + enc)
-
-        ret = hmac.hexdigest() + iv.hex() + enc.hex()
-    elif encryption_algorithm == PipelineD.HEConfig.AES256_ECB_HMAC_MD5:
-        key_val = key
-        key_mac = mac
+        return cipher.encrypt(s.encode('utf-8')).hex()
```
[the remark this answers is not in the document]
Comment (ajahl) [reply lte/gateway/python/magma/pipelined/encoding.py:33]: I created an issue #13193.

File: lte/gateway/python/magma/pipelined/encoding.py [hunk 2]
```diff
@@ -27,94 +27,80 @@
...
 
 
 def decrypt_str(data, key: bytes, encryption_algorithm, mac) -> str:
-    ret = ""
+ 
     if encryption_algorithm == PipelineD.HEConfig.RC4:
         cipher = ARC4.new(key)
-        ret = cipher.decrypt(data).hex()
-    elif encryption_algorithm == PipelineD.HEConfig.AES256_CBC_HMAC_MD5:
+        return cipher.decrypt(data).hex()
```
[the remark this answers is not in the document]
Comment (ajahl) [reply lte/gateway/python/magma/pipelined/encoding.py:63]: See above

File: lte/gateway/python/magma/pipelined/encoding.py [hunk 3]
```diff
@@ -27,94 +27,80 @@ def pad(m):
...
+            enc = aes_cipher.encrypt(pad(s).encode('utf-8'))
+            hmac.update(enc)
+            return gzip.compress(hmac.digest() + enc)
+
+    raise ValueError("Unsupported encryption algorithm")
 
 
 def decrypt_str(data, key: bytes, encryption_algorithm, mac) -> str:
-    ret = ""
+
```
Comment (sebathomas) [inline lte/gateway/python/magma/pipelined/encoding.py]: Nitpick: unnecessary empty line

**Review by sebathomas — APPROVED** [review]

**Comment by ajahl** [comment]
@magma/approvers-agw-pipelined, can someone take a look at this PR? Thanks!

**Review by nstng — APPROVED** [review]
LGTM (% NIT discussed offline, adding return types)

**Pull request merged**